subsystem-drive/implementations/steer-modes.hpp

```diff
@@ -22,5 +45,16 @@ namespace sjsu::drive
             // TODO: Calculate tri_wheel_router_arguments
             return {};
         }
+
+        // Helper Functions
+        ///<penis>
+        float CalculateAckerman(float inner_wheel_angle)
+        {
+            float outter_wheel_angle =
+                float(0.392 + 0.744 * abs(int(inner_wheel_angle)) +
+                      -0.0187 * pow(abs(int(inner_wheel_angle)), 2) +
+                      1.84E-04 * pow(abs(int(inner_wheel_angle)), 3));
+            return (inner_wheel_angle > 0) ? outter_wheel_angle : -outter_wheel_angle;
+        }
```

Comment:
Change the /// comment and move this to private. No need for auxiliary functions to be public.

Reply:
Good job otherwise, looks good.